app/src/main/java/org/jellyfin/mobile/downloads/DownloadItem.kt

```diff
@@ -4,13 +4,13 @@ import android.graphics.Bitmap
 import android.graphics.BitmapFactory
 import kotlinx.serialization.json.Json
 import org.jellyfin.mobile.data.entity.DownloadEntity
-import org.jellyfin.mobile.player.source.JellyfinMediaSource
+import org.jellyfin.mobile.player.source.RemoteJellyfinMediaSource
 import org.jellyfin.mobile.utils.Constants
 import java.io.File
 import java.util.Locale
 
 data class DownloadItem(private val download: DownloadEntity) {
-    val mediaSource: JellyfinMediaSource = Json.decodeFromString(download.mediaSource)
+    val mediaSource: RemoteJellyfinMediaSource = Json.decodeFromString(download.mediaSource)
```

Comment:
Same. There are othere sources were I do not understand the use of RemoteJellyfinMediaSource, where I believe it should be either LocalJellyfinMediaSource or JellyfinMediaSource